tests/rptest/tests/rpk_cluster_test.py

```diff
@@ -288,8 +277,19 @@ def obtain_license():
             'license_violation': False,
             'enterprise_features_in_use': [],
         }
-        result = json.loads(rp_license)
-        assert expected_license == result, result
+
+        def compare_license():
+            license = self._rpk.license_info()
```

Comment:
is the license here logged into the debug log so that if test fails someone can see what part of the license is not comparing equal to what is expected?

Reply:
that's a fair point. I'd probably just log it to debug, indiscriminately, right here. esp if it usually succeeds on the first go.

Reply:
Yes, we print it like:
```
[DEBUG - 2024-10-17 18:55:44,159 - rpk - _execute - lineno:1156]: Executing command: ['/var/lib/buildkite-agent/builds/buildkite-amd64-xfs-builders-i-0bbe49a4642636409-1/redpanda/redpanda/vbuild/redpanda_installs/ci/bin/rpk', '--api-urls', 'docker-rp-10:9644,docker-rp-21:9644,docker-rp-16:9644', 'cluster', 'license', 'info', '--format', 'json', '-v']
[DEBUG - 2024-10-17 18:55:44,208 - rpk - _execute - lineno:1176]: 
{"license_status":"not_present","organization":"redpanda-testing","type":"enterprise","expires":"Jul 11 2122","expires_unix":4813252273,"checksum_sha256":"...","license_expired":false,"license_violation":false,"enterprise_features_in_use":[]}
```